spec/system/show_game_rooms_spec.rb

```diff
@@ -25,10 +25,10 @@
       click_on room.name
     end
 
-    it 'tells user if there are no tracks on the room' do
+    it 'does not display rooms without tracks' do
       RoomTrack.find_by(room_id: room.id).destroy
-      visit room_path(room.id)
-      expect(page).to have_content 'Room doesn\'t exist.'
+      visit rooms_path
+      expect(page).not_to have_content room.name
     end
 
```

Comment:
Just a minor suggestion :) 
Can we put this in a separate context? To clearly distinguish what scenario we are testing compared to other specs. For example:
```ruby
context 'when user is logged in and joins a room without tracks' do
  it 'does not display room' do
    ...
  end
end
```

Reply:
Great suggestion! Thanks, @annagabriel-hash HYG! 

https://github.com/lea927/drop-that-beat/pull/83/commits/bb489548c0d2a4f116b65e65db8c049debf66724